.azurepipelines/BaseTools-Build-For-Publication.yml

```diff
@@ -97,10 +105,10 @@ jobs:
         filename: stuart_update
         arguments: -c .pytool/CISettings.py
 
-    - template: templates/basetools-build-steps.yml
+    - template: Steps/BuildBaseTools.yml@mu_devops
       parameters:
-        tool_chain_tag: 'VS2019'
         extra_parameters: '--skip_path_env'
+        tool_chain_tag: 'VS2019'
```

Comment:
is the 2019 intentional for base tools build?

Reply:
I just left it the way it was. This file needs additional follow up, but I wanted to save that for the future given the other work needed at the moment.